tree/tree/src/TTree.cxx

```diff
@@ -1060,6 +1061,17 @@ TTree::~TTree()
    }
 }
 
+TObject *TTree::Clone(const char *) const
+{
+   throw std::runtime_error("TObject::Clone() is not supported by TTree! Use TTree::CloneTree().");
```

Comment:
Minor: I think this is the first exception thrown in this file :)

Reply:
I agree with Philippe and I think it shouldn't be implemented this way, rather via

```
Error("Clone", "Not implemented, use XXXXXXX instead.");
```